src/main/java/de/tum/in/www1/artemis/service/StatisticsService.java

```diff
@@ -138,6 +146,7 @@ public CourseManagementStatisticsDTO getCourseStatistics(Long courseId) {
         var includedExercises = exercises.stream().filter(Exercise::isCourseExercise)
                 .filter(exercise -> !exercise.getIncludedInOverallScore().equals(IncludedInOverallScore.NOT_INCLUDED)).collect(Collectors.toSet());
         Double averageScoreForCourse = participantScoreRepository.findAvgScore(includedExercises);
+        averageScoreForCourse = averageScoreForCourse != null && averageScoreForCourse > 0.0 ? averageScoreForCourse : 0.0;
```

Comment:
```suggestion
        averageScoreForCourse = Objects.requireNonNullElse(averageScoreForCourse)
        averageScoreForCourse = Math.min(averageScoreForCourse, 0.0)
```
When would the average score be below `0.0`?